packages/message/src/lib/__tests__/message-service.spec.ts

```diff
@@ -0,0 +1,305 @@
+import { eventBus } from '@adorsys-gis/event-bus';
+import {
+  ServiceResponse,
+  ServiceResponseStatus,
+} from '@adorsys-gis/status-service';
+import { v4 as uuidv4 } from 'uuid';
+import { Message } from '../../model/Message';
+import { MessageEventChannel } from '../../model/MessageEventChannel';
+import { MessageService } from '../MessageService';
+
+describe('MessageService', () => {
+  let messageService: MessageService;
```

Comment:
I don't know what it is going wrong locally, but I can't run the test suite.

![image](https://github.com/user-attachments/assets/5d8f5ff5-8493-426e-8756-20c56a0ca832)

Reply:
Can you fix the conflict on the branch and try again locally?